docs/products/opensearch/concepts/indices.rst

```diff
@@ -46,14 +46,17 @@ These are only rough suggestions; the optimal values depend heavily on how you u
 Performance impact
 -------------------
 
-Having a large number of indices or shards affects the performance of your OpenSearch service. As an example, here are some figures from a three-node Aiven for OpenSearch business-8 cluster:
+Having a large number of indices or shards affects the performance of your OpenSearch service.  
+20 shards or few per GB of heap memory should be set as per `recommendation <https://www.elastic.co/guide/en/elasticsearch/reference/current/size-your-shards.html#shard-count-recommendation>`_.  As an example, an OpenSearch business-8 cluster comes with 8GB heap,
+20 x 8 = 160 or less would be the recommended configuration for number of shards.
 
```

Comment:
Would it make sense to include the shard size recommendations [1] as well? It think it would be valuable. 

[1] https://www.elastic.co/guide/en/elasticsearch/reference/current/size-your-shards.html#shard-size-recommendation

Reply:
Yes, thanks for your suggestion.  Shard size recommendation added!